psutil/_psutil_common.c

```diff
@@ -320,6 +316,20 @@ psutil_loadlibs() {
     // minumum requirement: Win 7
     GetLogicalProcessorInformationEx = psutil_GetProcAddressFromLib(
         "kernel32", "GetLogicalProcessorInformationEx");
+    // minimum requirements: Windows Server Core
+#ifdef UNICODE
```

Comment:
Mmm. Is this necessary? Is it possible for a system to not have unicode support?
Regardless from that, we already use other unicode "W" functions in many other places. If this is broken then they are all broken (in which case it should be a separate fix).